gui/default/assets/lang/lang-en.json

```diff
@@ -128,6 +128,7 @@
    "Error": "Error",
    "External File Versioning": "External File Versioning",
    "Failed Items": "Failed Items",
+   "Failed to load file versions.": "Failed to load file versions.",
```

Comment:
These seem to not get picked up automatically by `go run build.go translate`.  Could you check why?

Reply:
Never mind. They are picked up. My bad.